.github/workflows/release-nightly.yaml

```diff
@@ -0,0 +1,45 @@
+name: nightly release
+
+on: push
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
```suggestion
        uses: actions/checkout@v3
```

Reply:
Resolved in [`2fb6c78`](https://github.com/openfga/openfga/pull/508/commits/2fb6c78bb9703cf38416845979155b93c92bc761)